src/Microsoft.Framework.Runtime.Hosting/Json/JsonValue.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Framework.Runtime.Json
+{
+    internal class JsonValue
+    {
+        public JsonValue(JsonPosition position)
```

Comment:
They serves different purpose. 
`JsonToken` is the value returned by `JsonBuffer` it's the data interface between `JsonBuffer` and `JsonDeserializer`. It's a value type.

`JsonValue` is the base class of return values of `JsonDeserializer`, it is carries position information to form a `FileFormatException`. As base class it doesn't carry the value not it knows the type of the value.